etc/ui-config/mh_default_org/paella7/custom_theme_example/theme.json

```diff
@@ -0,0 +1,7 @@
+{
+    "styleSheets": [
+        "theme.css"
```

Comment:
There is a folder called `custom_theme_example` but also (Maybe left from another PR) `custom_theme.css`

Reply:
custom_theme_example is just an example. To use it you need to change in the config.json file the `opencast.theme` property to `custom_theme_example`.

`

Reply:
Now I checked between the compiled binary and the source code. In the binary, there is an additional file
 `custom_theme.css` that is outside the folder.
 
![image](https://github.com/opencast/opencast/assets/8040628/8fb75ef3-7a10-4ead-94eb-703f53520861)

If the custom themes should be only in folders, what is that `CSS` file?

Reply:
An old file. make a clean. That file should not be there.

Reply:
Strange, I did a `mvn  clean install.` I will reset the folder and then a new build